### ISSUE TITLE
jar can not start with v2.6.6

### ISSUE THREAD

**Issue by AlexStocks** [apache/incubator-dubbo#3777]
- [ ] I have searched the [issues](https://github.com/apache/incubator-dubbo/issues) of this repository and believe that this is not a duplicate.
- [ ] I have checked the [FAQ](https://github.com/apache/incubator-dubbo/blob/master/FAQ.md) of this repository and believe that this is not a duplicate.

### Environment

* Dubbo version: 2.6.6
* Operating System version: MacOS 10.14.3
* Java version: 1.8.0_151

### Steps to reproduce this issue

1. sh bin/start.sh

### Expected Result

The dubbo jar can not start.

### Actual Result

```
sh bin/start.sh
: No such file or directoryin
: No such file or directory.
: command not found7:
: command not found13:
bin/start.sh: line 93: syntax error: unexpected end of file
```

**Comment by lovepoem** [comment]
I checked the ` dubbo/dubbo-container/dubbo-container-api/src/main/resources/META-INF/assembly/bin/start.sh` file  on the 2.6.x branch ： 

Found nothing is changed recently 
![image](https://user-images.githubusercontent.com/827112/55314174-255d1780-549c-11e9-99ab-3688471311d3.png)

>  The dubbo jar can not start.

Which jar are you using?

**Comment by AlexStocks** [comment]
The format of start.sh is dos. After I formated it by command "dos2unix ./bin/start.sh", it can work now.

**Comment by AlexStocks** [comment]
@cvictory Has this bug fixed?

**Comment by cvictory** [comment]
> @cvictory Has this bug fixed? You can reproduce this issue easily by
> [java-server.zip](https://github.com/apache/incubator-dubbo/files/3041442/java-server.zip).

This shell is for linux bash,  the mac bash is different from that .
So can you test this demo in linux machine.

I will list some difference between mac and linux:
1. cd "$(dirname "$0")"  it work well in linux
but in mac , you should use  cd "$(dirname "$0")"
https://stackoverflow.com/questions/1308823/os-x-bash-dirname/1308838#1308838

2. it is also happened in pwd and other command.

**Comment by AlexStocks** [comment]
@cvictory 
![image](https://user-images.githubusercontent.com/7959374/55530983-fd0f2c00-56da-11e9-89ff-3cf6abc7af0f.png)

**Comment by cvictory** [comment]
Maybe I am wrong. Firstly I think it is different os problem, and i close this issue.
 I will check with 2.6.6 release manager next week.

If you have some solution to fix this , you can fire a pr and reopen this issue.

**Comment by AlexStocks** [comment]
@cvictory Firstly, I do not have the right to reopen this issue. Secondly, I review the start.sh in the github and I can not find any ^M symbol. So I do not know what's wrong in the release process.

**Comment by biyuhao** [comment]
hi @AlexStocks  , where you got the `java-server.zip`?
If you built the binary package yourself, maybe you can check your git config about `core.autocrlf`

**Comment by AlexStocks** [comment]
@biyuhao This package is my dubbo test codes. The start.sh v2.6.6 is from apache central maven repo, I think.

**Comment by biyuhao** [comment]
Hi, I've confirmed that both `dubbo-2.6.6.jar` and `dubbo-2.6.6-sources.jar` in [Maven Central](http://central.maven.org/maven2/com/alibaba/dubbo/2.6.6/) have the malformed scripts.
line endings should be `0x0a` not `0x0d0a`
![image](https://user-images.githubusercontent.com/1197310/55551531-a242f800-570d-11e9-8fde-9239f881ce24.png)

**Comment by biyuhao** [comment]
Seems release packages in [download page](http://dubbo.apache.org/en-us/blog/download.html) are not affected.
This will be fixed in next patch release, maybe `2.6.7`?

**Comment by AlexStocks** [comment]
@biyuhao Good job. Do u know what's wrong with the dubbo-2.6.6.jar that causes this script has "\r\n"?
